dubbo-common/src/main/java/org/apache/dubbo/common/logger/slf4j/Slf4jLoggerAdapter.java

```diff
@@ -45,6 +55,7 @@ public Level getLevel() {
 
     @Override
     public void setLevel(Level level) {
+        logger.warn("", "", "", String.format("The level of slf4j logger current can not be set, using the default level: %s", Slf4jLogger.getLevel(ROOT_LOGGER)));
```

Comment:
The first parameter should be `X-X`

Reply:
I know,but I have not find a good solution,may be 7-6?